src/protocol/http/src/main/java/org/apache/jmeter/protocol/http/curl/BasicCurlParser.java

```diff
@@ -107,17 +108,17 @@
     private static final int VERBOSE_OPT                = 'v';// $NON-NLS-1$
     private static final int SILENT_OPT                 = 's';// $NON-NLS-1$
 
-    private static final List<Integer> AUTH_OPT                 = Arrays.asList(BASIC_OPT, DIGEST_OPT);// $NON-NLS-1$
-    private static final List<Integer> SSL_OPT                  = Arrays.asList(CAFILE_OPT, CAPATH_OPT, CERT_OPT, CIPHERS_OPT,
-            CERT_STATUS_OPT, CERT_TYPE_OPT, KEY_OPT, KEY_TYPE_OPT);// $NON-NLS-1$
-    private static final List<Integer> DATAS_OPT                = Arrays.asList(DATA_OPT, DATA_ASCII_OPT, DATA_BINARY_OPT,
-            DATA_URLENCODE_OPT, DATA_RAW_OPT);// $NON-NLS-1$
-    private static final List<Integer> FORMS_OPT                = Arrays.asList(FORM_OPT, FORM_STRING_OPT);// $NON-NLS-1$
-    private static final List<Integer> IGNORE_OPTIONS_OPT       = Arrays.asList(OUTPUT_OPT, CREATE_DIRS_OPT, RAW_OPT,
-            INCLUDE_OPT, KEEPALIVETILE_OPT, VERBOSE_OPT, SILENT_OPT);// $NON-NLS-1$
-    private static final List<Integer> NOSUPPORT_OPTIONS_OPT    = Arrays.asList(PROXY_NTLM_OPT, PROXY_NEGOTIATE_OPT);// $NON-NLS-1$
-    private static final List<Integer> PROPERTIES_OPT           = Arrays.asList(MAX_REDIRS_OPT);// $NON-NLS-1$
-    private static final List<String> DYNAMIC_COOKIES           = Arrays.asList("PHPSESSID", "JSESSIONID", "ASPSESSIONID",
+    private static final List<Integer> AUTH_OPT              = Arrays.asList(BASIC_OPT, DIGEST_OPT);
+    private static final List<Integer> SSL_OPT               = Arrays.asList(CAFILE_OPT, CAPATH_OPT, CERT_OPT, CIPHERS_OPT,
+            CERT_STATUS_OPT, CERT_TYPE_OPT, KEY_OPT, KEY_TYPE_OPT);
+    private static final List<Integer> DATAS_OPT             = Arrays.asList(DATA_OPT, DATA_ASCII_OPT, DATA_BINARY_OPT,
+            DATA_URLENCODE_OPT, DATA_RAW_OPT);
+    private static final List<Integer> FORMS_OPT             = Arrays.asList(FORM_OPT, FORM_STRING_OPT);
+    private static final List<Integer> IGNORE_OPTIONS_OPT    = Arrays.asList(OUTPUT_OPT, CREATE_DIRS_OPT, RAW_OPT,
+            INCLUDE_OPT, KEEPALIVETILE_OPT, VERBOSE_OPT, SILENT_OPT);
+    private static final List<Integer> NOSUPPORT_OPTIONS_OPT = Arrays.asList(PROXY_NTLM_OPT, PROXY_NEGOTIATE_OPT);
```

Comment:
@pmouawad , @ham1 , please refrain committing changes that try to keep horizontal layout of variable declarations.
It does not help readability that much, however, it provokes merge-conflicts, and it makes it harder to analyze Git history (the lines are shown as changed for no reason)

See https://google.github.io/styleguide/javaguide.html#s4.6.3-horizontal-alignment

Reply:
This also removed the no longer required comments and then, rather than leaving them misaligned, aligned them. I agree that alignment isn't good as it doesn't aid readability but it does add maintenance (or they become misaligned).

Reply:
It seems I failed to convey the message.
Please **never** try to align entries.